addons/stock/migrations/13.0.1.1/post-migration.py

```diff
@@ -146,7 +146,7 @@ def fill_propagate_date_minimum_delta(env):
         FROM ir_config_parameter icp
         WHERE sr.propagate_date IS NULL
             AND icp.key = 'stock.use_propagation_minimum_delta'
-            AND icp.value != 'True'"""
+            AND icp.value::boolean IS DISTINCT FROM TRUE"""
```

Comment:
Or it can be `NULL`?
```suggestion
            AND NOT icp.value::boolean"""
```

Reply:
Yeah, right. Theoretically, it can be null if it's set with set_param(key, NULL), but that never happens haha